### PR TITLE
[IMP]10716 add module report_printout_improvement.

### PR DESCRIPTION
**Pull request by Rona111**

### CODE REVIEW

File: report_printout_improvement/__init__.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
```
Comment (elicoidal) [inline report_printout_improvement/__init__.py]: headers/readme/openerp.py

**Comment by Rona111** [comment]
cc @elicoidal @siyuancn @duanyp1991 @seb-elico

**Comment by elicoidal** [comment]
It should be one module for SO, one for PO and one for stock.
names: sale_printout_sequence, purchase_printout_sequence, stock_picking_sequence

**Comment by Rona111** [comment]
cc @elicoidal @siyuancn @duanyp1991 @seb-elico

File: purchase_printout_sequence/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# © 2015 Elico Corp (www.elico-corp.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': 'Purchase Picking Sequence',
+    'version': '8.0.1.0.1',
+    'depends': [
+        'stock',
```
Comment (siyuancn) [inline purchase_printout_sequence/__openerp__.py]: why depends on stock?

File: purchase_printout_sequence/README.rst [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+=========================
+
+This module extends the functionality of report to This module adds the sequence and total quantity into the purchase order printout.
+
+Installation
+============
+
+To install this module, you need to:
+
+ * have basic modules installed (stock)
```
Comment (siyuancn) [inline purchase_printout_sequence/README.rst]: why stock?

**Comment by elicoidal** [comment]
LGTM (no technical test/check)

File: sale_printout_sequence/README.rst [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+=====================
+
+This module extends the functionality of report to This module adds the sequence and total quantity into the sales order printout.
+
+Installation
+============
+
+To install this module, you need to:
+
+ * have basic modules installed (stock)
```
Comment (siyuancn) [inline sale_printout_sequence/README.rst]: why stock?

File: sale_printout_sequence/__openerp__.py [hunk 5]
```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# © 2015 Elico Corp (www.elico-corp.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': 'Sale Picking Sequence',
+    'version': '8.0.1.0.1',
+    'depends': [
+        'stock',
```
Comment (siyuancn) [inline sale_printout_sequence/__openerp__.py]: wrong module

File: purchase_printout_sequence/README.rst [hunk 6]
```diff
@@ -0,0 +1,52 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=========================
+Purchase Picking Sequence
+=========================
+
+This module extends the functionality of report to This module adds the sequence and total quantity into the purchase order printout.
```
Comment (siyuancn) [inline purchase_printout_sequence/README.rst]: This module extends the functionality of report to adds the sequence and total quantity into the purchase order printout.

File: sale_printout_sequence/README.rst [hunk 7]
```diff
@@ -0,0 +1,53 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=====================
+Sale Picking Sequence
+=====================
+
+This module extends the functionality of report to This module adds the sequence and total quantity into the sales order printout.
```
Comment (siyuancn) [inline sale_printout_sequence/README.rst]: This module extends the functionality of report to adds the sequence and total quantity into the sales order printout.

File: stock_picking_sequence/README.rst [hunk 8]
```diff
@@ -0,0 +1,53 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+======================
+Stock Picking Sequence
+======================
+
+This module extends the functionality of report to This module adds the sequence and total quantity into the delivery order printout.
```
Comment (siyuancn) [inline stock_picking_sequence/README.rst]: This module extends the functionality of report to adds the sequence and total quantity into the delivery order printout.

**Comment by Rona111** [comment]
cc @elicoidal @siyuancn fix the dependency.

**Comment by elicoidal** [comment]
LGTM

**Comment by siyuancn** [comment]
LGTM

**Pull request merged**